_quarto.yml

```diff
@@ -47,7 +47,7 @@ website:
         text: AFSC BSAI Atka Mackerel case study
       - href: content/SWFSC-sardine.qmd
         text: SWFSC sardine case study
-      - href: content/PIFS-opakapaka.qmd
+      - href: content/PIFSC-opaka-length-M2.qmd
```

Comment:
I would like to change this to PIFSC-opakapaka.qmd and remove the old qmd file in favor of the new length-based qmd file. Just one version of opakapaka going forward is best I think.